pytorch_toolkit/human_pose_estimation/README.md

```diff
@@ -79,62 +100,79 @@ We observe this error with maximum number of open files (`ulimit -n`) equals to
+    ```
+    This produces the `human-pose-estimation.xml` model and weights `human-pose-estimation.bin` in single-precision floating-point format (FP32).
 
-## Output format <a name="output-format"/>
+## Output Format
 
-Network has two outputs:
+The network has two outputs:
 * Keypoints heatmaps
-* Part affinity fileds (used in grouping by persons instances)
+* Part affinity fields (used in grouping by persons instances)
 
-These outputs are post-processed, and [`group_keypoints`](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/human_pose_estimation/modules/keypoints.py#L51) function returns found poses, represented as two lists:
-* `all_keypoints` - list of keypoints for all poses found in image. Each element is a keypoint with its confidence and global id, represented as tuple with 4 elements: 0, 1 - keypoints coordinates, 2 - keypoint confidence (value from heatmap) and the last one is global keypoint id in this list.
-* `pose_entries` - list of poses. Each pose is a list of 20 elements: [0,17] - global keypoints ids, 18 is pose confidence (sum of keypoints confidences and confidences of connections between keypoints), the last one is number of found keypoints for pose.
+These outputs are post-processed, and [`group_keypoints`](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/human_pose_estimation/modules/keypoints.py#L51) function returns recognized poses, represented as two lists:
+* `all_keypoints` - list of keypoints for all poses found in an image. Each element is a keypoint with its confidence and global ID, represented as a tuple with 4 elements: 0, 1 - keypoints coordinates, 2 - keypoint confidence (value from heatmap) and the last one is the global keypoint id in this list.
+* `pose_entries` - list of poses. Each pose is a list of 20 elements: [0,17] - global keypoints IDs, 18 is the pose confidence (sum of keypoints confidences and confidences of connections between keypoints), the last one is the number of recognized keypoints for the pose.
 
-Keypoints order and names are defined in the [Pose](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/human_pose_estimation/modules/pose.py#L7) class. Example of parsing is shown in [demo](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/human_pose_estimation/demo.py#L112).
+Keypoints order and names are defined in the [Pose](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/human_pose_estimation/modules/pose.py#L7) class. Example of parsing is shown in the [demo](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/human_pose_estimation/demo.py#L112).
 
-## C++ Demo <a name="cpp-demo"/>
+## <a name="cpp-demo"/>C++ Demo</a>
```

Comment:
Why this style is better? It was shorter before.

Reply:
In my editor it looked broken, although i haven't checked the html output